bitnami/metallb/values.yaml

```diff
@@ -167,6 +203,18 @@ controller:
     successThreshold: 1
     timeoutSeconds: 1
 
+  prometheus:
+    # Prometheus Operator service monitors
```

Comment:
Could you follow the comments standard here like:
```
## Description
##
parameter: value
```

Reply:
@miguelaeh

I tried autogenerating the markdown Table from the `values.yaml` file in the chart.
The ouput was ok. The descriptions are just pain.

[https://gist.github.com/Cellebyte/ec629603952a98be1d7dbcd9b7aac3f4](https://gist.github.com/Cellebyte/ec629603952a98be1d7dbcd9b7aac3f4)

# values.yaml
|                       Parameter                        |                                                                    Description                                                                     |                Default                |
|--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
| rbac.create                                            | create specifies whether to install and use RBAC rules.                                                                                            | True                                  |
| psp.create                                             | create specifies whether to install Pod Security Policies.                                                                                         | False                                 |
| prometheusRule.enabled                                 |                                                                                                                                                    | False                                 |
| controller.image.registry                              |                                                                                                                                                    | docker.io                             |
| controller.image.repository                            |                                                                                                                                                    | bitnami/metallb-controller            |
| controller.image.tag                                   |                                                                                                                                                    | 0.9.3-debian-10-r156                  |
| controller.image.pullPolicy                            |                                                                                                                                                    | IfNotPresent                          |
| controller.rbac.create                                 | create specifies whether to install and use RBAC rules.                                                                                            | True                                  |
| controller.psp.create                                  | create specifies whether to install Pod Security Policies.                                                                                         | True                                  |
| controller.tolerations                                 |                                                                                                                                                    | []                                    |
| controller.podAffinityPreset                           |                                                                                                                                                    |                                       |
| controller.podAntiAffinityPreset                       |                                                                                                                                                    | soft                                  |
| controller.nodeAffinityPreset.type                     | Node affinity type Allowed values: soft, hard                                                                                                      |                                       |
| controller.nodeAffinityPreset.key                      | Node affinity type Allowed values: soft, hard                                                                                                      |                                       |
| controller.nodeAffinityPreset.values                   | Node affinity type Allowed values: soft, hard                                                                                                      | []                                    |
| controller.serviceAccount.create                       | Specifies whether a ServiceAccount should be created                                                                                               | True                                  |
| controller.serviceAccount.name                         | Specifies whether a ServiceAccount should be created                                                                                               |                                       |
| controller.securityContext.enabled                     |                                                                                                                                                    | True                                  |
| controller.securityContext.runAsNonRoot                |                                                                                                                                                    | True                                  |
| controller.securityContext.runAsUser                   |                                                                                                                                                    |                                  1001 |
| controller.securityContext.fsGroup                     |                                                                                                                                                    |                                  1001 |
| controller.securityContext.allowPrivilegeEscalation    |                                                                                                                                                    | False                                 |
| controller.securityContext.readOnlyRootFilesystem      |                                                                                                                                                    | True                                  |
| controller.securityContext.capabilities.drop           |                                                                                                                                                    | ['ALL']                               |
| controller.revisionHistoryLimit                        |                                                                                                                                                    |                                     3 |
| controller.terminationGracePeriodSeconds               |                                                                                                                                                    |                                     0 |
| controller.containerPort.metrics                       |                                                                                                                                                    |                                  7472 |
| controller.livenessProbe.enabled                       |                                                                                                                                                    | True                                  |
| controller.livenessProbe.failureThreshold              |                                                                                                                                                    |                                     3 |
| controller.livenessProbe.initialDelaySeconds           |                                                                                                                                                    |                                    10 |
| controller.livenessProbe.periodSeconds                 |                                                                                                                                                    |                                    10 |
| controller.livenessProbe.successThreshold              |                                                                                                                                                    |                                     1 |
| controller.livenessProbe.timeoutSeconds                |                                                                                                                                                    |                                     1 |
| controller.readinessProbe.enabled                      |                                                                                                                                                    | True                                  |
| controller.readinessProbe.failureThreshold             |                                                                                                                                                    |                                     3 |
| controller.readinessProbe.initialDelaySeconds          |                                                                                                                                                    |                                    10 |
| controller.readinessProbe.periodSeconds                |                                                                                                                                                    |                                    10 |
| controller.readinessProbe.successThreshold             |                                                                                                                                                    |                                     1 |
| controller.readinessProbe.timeoutSeconds               |                                                                                                                                                    |                                     1 |
| controller.prometheus.serviceMonitor.enabled           | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | False                                 |
| controller.prometheus.serviceMonitor.jobLabel          | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | app.kubernetes.io/name                |
| controller.prometheus.serviceMonitor.interval          | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       |                                       |
| controller.prometheus.serviceMonitor.metricRelabelings | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | []                                    |
| controller.prometheus.serviceMonitor.relabelings       | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | []                                    |
| speaker.image.registry                                 |                                                                                                                                                    | docker.io                             |
| speaker.image.repository                               |                                                                                                                                                    | bitnami/metallb-speaker               |
| speaker.image.tag                                      |                                                                                                                                                    | 0.9.3-debian-10-r166                  |
| speaker.image.pullPolicy                               |                                                                                                                                                    | IfNotPresent                          |
| speaker.rbac.create                                    | create specifies whether to install and use RBAC rules.                                                                                            | True                                  |
| speaker.psp.create                                     | create specifies whether to install Pod Security Policies.                                                                                         | True                                  |
| speaker.tolerations                                    |                                                                                                                                                    | []                                    |
| speaker.serviceAccount.create                          | Specifies whether a ServiceAccount should be created                                                                                               | True                                  |
| speaker.serviceAccount.name                            | Specifies whether a ServiceAccount should be created                                                                                               |                                       |
| speaker.daemonset.terminationGracePeriodSeconds        | Configure the grace time period for sig term ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/hook-handler-execution  |                                     2 |
| speaker.daemonset.hostPorts.metrics                    | Configure the grace time period for sig term ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/hook-handler-execution  |                                  7472 |
| speaker.securityContext.enabled                        |                                                                                                                                                    | True                                  |
| speaker.securityContext.runAsUser                      |                                                                                                                                                    |                                     0 |
| speaker.securityContext.allowPrivilegeEscalation       |                                                                                                                                                    | False                                 |
| speaker.securityContext.readOnlyRootFilesystem         |                                                                                                                                                    | True                                  |
| speaker.securityContext.capabilities.drop              |                                                                                                                                                    | ['ALL']                               |
| speaker.securityContext.capabilities.add               |                                                                                                                                                    | ['NET_ADMIN', 'NET_RAW', 'SYS_ADMIN'] |
| speaker.extraEnvVars                                   |                                                                                                                                                    | []                                    |
| speaker.livenessProbe.enabled                          |                                                                                                                                                    | True                                  |
| speaker.livenessProbe.failureThreshold                 |                                                                                                                                                    |                                     3 |
| speaker.livenessProbe.initialDelaySeconds              |                                                                                                                                                    |                                    10 |
| speaker.livenessProbe.periodSeconds                    |                                                                                                                                                    |                                    10 |
| speaker.livenessProbe.successThreshold                 |                                                                                                                                                    |                                     1 |
| speaker.livenessProbe.timeoutSeconds                   |                                                                                                                                                    |                                     1 |
| speaker.readinessProbe.enabled                         |                                                                                                                                                    | True                                  |
| speaker.readinessProbe.failureThreshold                |                                                                                                                                                    |                                     3 |
| speaker.readinessProbe.initialDelaySeconds             |                                                                                                                                                    |                                    10 |
| speaker.readinessProbe.periodSeconds                   |                                                                                                                                                    |                                    10 |
| speaker.readinessProbe.successThreshold                |                                                                                                                                                    |                                     1 |
| speaker.readinessProbe.timeoutSeconds                  |                                                                                                                                                    |                                     1 |
| speaker.prometheus.serviceMonitor.enabled              | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | False                                 |
| speaker.prometheus.serviceMonitor.jobLabel             | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | app.kubernetes.io/name                |
| speaker.prometheus.serviceMonitor.interval             | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       |                                       |
| speaker.prometheus.serviceMonitor.metricRelabelings    | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | []                                    |
| speaker.prometheus.serviceMonitor.relabelings          | Prometheus Operator service monitors  enable support for Prometheus Operator                                                                       | []                                    |